spot_wrapper/testing/helpers.py

```diff
@@ -3,20 +3,18 @@
 import functools
 import typing
 
-import grpc
-from bosdyn.api.header_pb2 import CommonError
 from bosdyn.api.lease_pb2 import ResourceTree
 
 
-class GeneralizedDecorator:
+class ThinDecorator:
```

Comment:
Any chance we can get a more descriptive name?

Reply:
I'm not always great with names. `TransparentDecorator`? `ForwardingDecorator` is almost tautological.

Reply:
Naming might be the hardest thinking in programming, but I can't tell you what a `ThinDecorator` does just by reading the name. Maybe just add a usage comment?

Reply:
How about `ForwardingWrapper`? https://github.com/bdaiinstitute/spot_wrapper/pull/120/commits/6901afa781bdb47f1f739a5b0d867bf9a7cd6651